content/en/agent/basic_agent_usage/aix.md

```diff
@@ -80,14 +72,26 @@ Occasionally a proxy configuration must be specified depending on your network s
 
 ## Integrations
 
-Additional integrations available:
+The Unix Agent collects system metrics for:
+  
+* cpu
+* filesystem
+* iostat
+* load
+* memory
+* uptime
+* disk
+* network
+
+Additionally, the following integrations can be enabled to collect further metrics:
 
 * process
 * lparstats
-* disk
-* network
+* [ibm_was (Websphere Application Server)](https://github.com/DataDog/datadog-unix-agent/blob/master/checks/bundled/ibm_was/README.md)
```

Comment:
```suggestion
* [ibm_was (Websphere Application Server)][3]
```

Reply:
Then add this link to the bottom of the file as:

`[3]: https://github.com/DataDog/datadog-unix-agent/blob/master/checks/bundled/ibm_was/README.md`